examples/monorepo-package/packages/basic/doczrc.js

```diff
@@ -0,0 +1,4 @@
+export default {
+  menu: ['Getting Started', 'Components'],
+  repoRootPath: '../..',
```

Comment:
could we skip this by looking up the tree for a `package.json` instead? If you have something super special, I guess you could set this as a last resort - but stuff that just works is a great deal!

Reply:
Yeah I don't like it either and was really hesitant to add (yet another) config variable, but I couldn't think of a way to find the monorepo root.

> could we skip this by looking up the tree for a package.json instead?

We could, but how would we decide which parent directory to use ? 

For example for the following structure :

```
monorepo/
	package.json
	packages/
		basic/
			package.json
			.docz/
				package.json

```
The `basic` package could have its dependencies in basic/node_modules or monorepo/node_modules depending on the workspace configuration.

Any thoughts on a way around this ?

Reply:
🤔
We could check for the existence of the gatsby binary file - that's the only use we have for this (at least for now, right)? For the project root, we could just find the package.json furthest up the tree.

Reply:
It now looks for the first gatsby dependency up the tree and uses that .
I left `repoRootPath` in the config as an escape hatch, for people that have very particular setups.

Reply:
Amazing job 💪 remember to document it!